<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SukoMain</class>
 <widget class="QMainWindow" name="SukoMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>56</green>
        <blue>56</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>50</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>56</green>
        <blue>56</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>SukoMain</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PirateJoe13's Suko Solver</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="TopLeftButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="TopCenterButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="TopRightButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MiddleLeftButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MiddleCenterButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MiddleRightButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BottomRightButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BottomCenterButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BottomLeftButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="purpleLineEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kohinoor Bangla</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
border: 1px solid rgba(255, 255, 255, 128);
background-color: #885dff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="blueLineEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kohinoor Bangla</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
border: 1px solid rgba(255, 255, 255, 128);
background-color:rgb(96, 175, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pinkLineEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kohinoor Bangla</family>
      <pointsize>25</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
border: 1px solid rgba(255, 255, 255, 128);
background-color: rgb(199, 127, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Quadrant2">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>130</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kohinoor Bangla</family>
      <pointsize>35</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border: 2px solid rgb(183, 183, 183);
background-color:rgb(152, 152, 152);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Quadrant4">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>240</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kohinoor Bangla</family>
      <pointsize>35</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border: 2px solid rgb(183, 183, 183);
background-color:rgb(152, 152, 152);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Quadrant1">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>130</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kohinoor Bangla</family>
      <pointsize>35</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border: 2px solid rgb(183, 183, 183);
background-color:rgb(152, 152, 152);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Quadrant3">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>240</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kohinoor Bangla</family>
      <pointsize>35</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border: 2px solid rgb(183, 183, 183);
background-color:rgb(152, 152, 152);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BottomRightButton_2">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>439</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
border: 1px solid rgba(255, 255, 255, 128);
background-color: qlineargradient(spread:reflect, x1:1, y1:0.477273, x2:0, y2:0.488636, stop:0 rgba(85, 0, 255, 255), stop:0.502463 rgba(0, 189, 255, 255), stop:1 rgba(151, 0, 255, 255));</string>
    </property>
    <property name="text">
     <string>SOLVE</string>
    </property>
   </widget>
   <zorder>purpleLineEdit</zorder>
   <zorder>label</zorder>
   <zorder>TopLeftButton</zorder>
   <zorder>TopCenterButton</zorder>
   <zorder>TopRightButton</zorder>
   <zorder>MiddleLeftButton</zorder>
   <zorder>MiddleCenterButton</zorder>
   <zorder>MiddleRightButton</zorder>
   <zorder>BottomRightButton</zorder>
   <zorder>BottomCenterButton</zorder>
   <zorder>BottomLeftButton</zorder>
   <zorder>blueLineEdit</zorder>
   <zorder>pinkLineEdit</zorder>
   <zorder>Quadrant2</zorder>
   <zorder>Quadrant4</zorder>
   <zorder>Quadrant1</zorder>
   <zorder>Quadrant3</zorder>
   <zorder>BottomRightButton_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
